contracts/RMRK/extension/tokenProperties/RMRKPropertiesAccessControl.sol

```diff
@@ -0,0 +1,251 @@
+// SPDX-LicenseIdentifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+import "./IRMRKPropertiesAccessControl.sol";
+
+contract RMRKPropertiesAccessControl is IRMRKPropertiesAccessControl {
+    // mapping(address collection => mapping (ParameterType => mapping (uint256 parameterId => AccessControl))) private _accessControl;
+    mapping(address => mapping(ParameterType => mapping(uint256 => AccessControl)))
```

Comment:
No need for ParameterType here IMO.

Reply:
Agree, it will be removed from all of the access control 👌